packages/template-mrt-reference-app/app/ssr.js

```diff
@@ -67,12 +67,21 @@ const ENVS_TO_EXPOSE = [
     'aws_lambda_log_stream_name',
     'aws_region',
     'bundle_id',
+    // These "customer" defined environment variables are set by the Manager 
+    // and expected by the MRT smoke test suite
+    'customer_defined_array',
+    'customer_defined_boolean',
+    'customer_defined_float',
+    'customer_defined_integer',
+    'customer_defined_json',
+    'customer_defined_null',
+    'customer_defined_string',
     'deploy_id',
     'deploy_target',
     'external_domain_name',
     'mobify_property_id',
     'node_env',
-    'tz'
+    'tz',
```

Comment:
you probably need to run `npm run lint:fix` from the root of the monorepo for compliance with the comma dangle eslint setting

Reply:
Resolved in commit [2251e44](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/1207/commits/2251e4449ca4f4042f01a652ae0da78edb37eac7)